Mathlib/Algebra/Polynomial/Laurent.lean

```diff
@@ -513,39 +513,38 @@ instance isLocalization : IsLocalization.Away (X : R[X]) R[T;T⁻¹] :=
       exact ⟨1, rfl⟩ }
 
 theorem mk'_mul_T (p : R[X]) (n : ℕ) :
-  IsLocalization.mk' R[T;T⁻¹] p (⟨X^n, n, rfl⟩ : Submonoid.powers (X : R[X])) * T n =
-    toLaurent p := by
+    IsLocalization.mk' R[T;T⁻¹] p (⟨X^n, n, rfl⟩ : Submonoid.powers (X : R[X])) * T n =
```

Comment:
pre-existing: should there be a space after the semicolon?

Reply:
This file seems to consistently not put one, so I'll not touch that here